### DISCUSSION TITLE
Config with a `stackblitz` field in package.json

### DISCUSSION THREAD

**Discussion by patak-cat** [stackblitz/core#1760] [Ideas]
The original idea is from @antfu. In [Vitest](https://vitest.dev) we are using StackBlitz to add online playgrounds for the examples. See https://vitest.dev/guide/#examples.

In each example in https://github.com/vitest-dev/vitest/tree/main/examples, we need to add a `.stackblitzrc` file. This increases the cognitive load for new users because they need to understand this extra file. In some examples, a project may have only a handful of files so adding another one is not free.

An idea to avoid this issue would be to allow to configure the playground with a `stackblitz` field in `package.json`. This is standard for other tools like prettier, eslint, etc.

**Comment by SamVerschueren** [comment]
It would be awesome to have support for `stackblitz` property in `package.json`. However for now, I think you can get away with query params alone while still preserving the same functionality.

It seems that the `.stackblitzrc` in the examples is identical and always this

```json
{
  "installDependencies": true,
  "startCommand": "npm run test:ui"
}
```

`installDependencies` is `true` by default if there's a `package.json`, so that's already covered. The `startCommand` can be replaced with `?terminal=test:ui` in the query params.

This will not work if you use the "open from github" functionality where you just replace `github.com` with `github.stackblitz.com` in the examples directory. So for that to work, we need the `package.json#stackblitz` support.
